src/controllers/UsersController.php

```diff
@@ -386,6 +386,38 @@ public function deleteUser($request, $db)
         $view->setResponseCode(204);
     }
 
+    /**
+     * Allow users to be set as trusted
+     *
+     * @param $request Request
```

Comment:
The ```$request``` and ```$pdo``` are type-hinted in a very inconsistent way throughout the method-calls. So leaving them without a type-hint would be OK from my POV but we then should perhaps add an issue to JIRA to go through all the Controllers and change that.